ddtrace/contrib/urllib3/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+The ``urllib3`` integration instruments tracing on http calls with optional
+support for distributed tracing across services the client communicates with.
+
+The ``patch`` function must be called before importing and using the library.
+Note that instrumentation is turned off by default when using the ``patch_all``
+command and you must specifically patch urllib3 to enable ttracing.
+For example:
+
+    from ddtrace import patch
+    patch(urllib3=True)
+    import urllib3
+
+    http = urllib3.PoolManager()
+    r = http.request('GET', 'https://example.com/')
+
+    print(r.status)
+    print(r.data)
+
+The auto-instrumentation of requests can be further tuned by configuring
+urllib3 through the Configuration API:
+
+    from ddtrace import config
+
+    # disable distributed tracing
+    config.urllib3['distributed_tracing'] = False
```

Comment:
Why would anyone want to disable distributed tracing? is this a pattern we're using in other integrations?

Reply:
Yeah, this is common across integrations, but very rarely used.

This is mostly for, if I am only making API calls to third party services and I don't want to send the `X-Datadog-*` headers.